src/customizations/volto/components/manage/Blocks/Search/SearchBlockView.jsx

```diff
@@ -57,12 +60,36 @@ const applyDefaults = (data, root) => {
       v: root || '/',
     },
   ];
-  return {
+
+  const searchBySearchableText = data.query.filter(
+    (item) => item['i'] === 'SearchableText',
+  ).length;
+
+  const sort_on =
+    searchBySearchableText === 0
+      ? data?.sort_on
+        ? { sort_on: data.sort_on }
+        : { sort_on: 'effective' }
+      : undefined;
+
+  const sort_order =
+    searchBySearchableText === 0
+      ? data?.sort_order
+        ? { sort_order: data.sort_order }
+        : { sort_order: 'descending' }
+      : undefined;
```

Comment:
Giusto per capire: avete quindi concordato che se la ricerca è testuale viene ignorata completamente l'impostazione di ordinamento scelta dall'utente? Mi lascia perplesso, perché è una forzatura che non indichiamo all'utente. La soluzione che sta nel commit indicato nel commento sopra mi sembra più coerente con le scelte possibili che ha l'utente.

Reply:
Chiariti a voce, riassumo. Quello che si vuole è che nella ricerca iniziale vinca l'ordinamento che imposta il redattore. Nel momento in cui, però, si fa una ricerca testuale deve vincere l'ordinamento per ranking oppure il rischio è di avere una decina di risultati con quello più interessante che si trova in fondo alla lsita